README.md

```diff
@@ -66,7 +66,19 @@ compiled against the same MPI implementation used by
 configure [MPI.jl](https://github.com/JuliaParallel/MPI.jl) to not use the
 default MPI binary provided by JLL wrappers, you also need to build the C
 library [`p4est`](https://github.com/cburstedde/p4est) locally using the same
-MPI implementation. This is typically the situation on HPC clusters. If you
+MPI implementation. This is typically the situation on HPC clusters. 
+Note that in this case you need to set the configure option `--enable-shared`, 
+i.e., the [recommended production-run configuration](https://github.com/cburstedde/p4est/blob/2296a990d8b6b54731a63be0ba5bc17b08cd1f3d/README#L139-L140)
+should be changed to
+
+```bash
+/path/to/configure CFLAGS="-O2 -Wall -Wno-unused-parameter" --enable-mpi --enable-shared
+```
+Otherwise, the `make` and `make install` calls do not build the desired shared object file `libp4est.so` 
```

Comment:
I thought it might make sense to explicitly mention the `make` `make install` calls here as the (IMO very poor) `p4est` installation instructions do not mention these. Surely, for anyone who is familiar with the `configure make make install` process this is obvious, but if people come from Julia this is not obvious at all.

Reply:
Sounds good to me!

Reply:
I agree, thanks! I think I usually watched the relevant part of https://www.youtube.com/watch?v=jwvkujZ6lCA, when I wanted to recall how to build p4est locally :sweat_smile: